t/127-uthread-kill.t

```diff
@@ -507,3 +507,112 @@ thread created: zombie
 lua tcp socket abort resolver
 --- error_log
 
+=== TEST 10: non parent kill, parent waits for the killed
+--- config
+    location = /t {
+        content_by_lua '
+            function killing_thread(thread_to_kill)
+               ngx.sleep(1)
```

Comment:
@aviramc Can we use smaller sleeping intervals so that the tests can run much faster? Tests are meant to run over and over again, making them as fast as possible can be a big win :) I think times like 0.1 and 0.2 (seconds) should be long enough for our purposes.